tutorials/CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+
+ ### Contributor License Agreement
+
+ All third-party contributions to this project must be accompanied by a signed contributor license. This gives Adobe permission to redistribute your contributions as part of the project. Sign our [CLA](http://opensource.adobe.com/cla.html). You only need to submit an Adobe CLA one time, so if you have submitted one previously, you are probably good to go!
+
+ ### Code Reviews
+
+ All submissions should come in the form of pull requests and need to be reviewed by project committers. Read [GitHub's pull request documentation](https://help.github.com/articles/about-pull-requests/) for more information on sending pull requests.
+
+## 2. Sign Our CLA (Contributor License Agreement)
+
+In order to contribute to our documentation, you **MUST** sign the contributor license agreement, also known as the CLA. If you've signed and submitted one previously, you're good to go! Otherwise, visit [here](http://opensource.adobe.com/cla.html) to get signing.
+
+## 3. Clone the Repository
+
+1. Open your terminal and navigate to the place you want to store the project.
+2. Copy this command and paste it into the terminal.  
+```git clone https://github.com/Adobe-Marketing-Cloud/reactor-user-docs.git```
+3. Open the project in your editor. 
+
+## 4. Checkout to a New Branch
+
+In your terminal, run  
+```git checkout -b [name-of-your-branch]```
+
+## 5. Create Your Content  
+
+You are now set to begin creating your tutorial page. Be sure your new file title is descriptive, succinct, and is a .md file.  
+**Good title**: "creating-a-new-rule.md"   
+MD stands for markdown. If you are unfamiliar with .md files or markdown, you can learn more [here](https://guides.github.com/features/mastering-markdown/). A template.md file is located in the tutorials folder to help you get started. 
+
+## 6. Create a New Pull Request
+
+Once you've finished with your tutorial and are happy with the changes, it's time for a pull request! This is basically a request to have your changes added to the live site. It's important to note that any changes outisde of your new tutorial files will **not** be accepted.
+1. Open your terminal to the reactor-user-docs project. 
+2. Enter  
+ ```git add .```  
+ and press enter.
+3. Enter  
+```git commit -m "New tutorial"```  
+and press enter.
+4. Enter  
+```git push origin [name-of-your-branch]```
+5. Return to https://github.com/Adobe-Marketing-Cloud/reactor-user-docs. There should now be a green button that says "Create pull request". If there's not, click on the gray button that says "New pull request." You may choose to leave a comment, then click "Create pull request".
+You may receive requested changes after your pull request is reviewed. After those changes are made, follow the steps 1-4 again. Once everything is satisfactory, your pull request will be merged into the master branch.
```

Comment:
"You may receive requested changes..." is a bit unclear. Are you intending to say something like, "You may be asked to make changes..." If that's the case, then the next sentence might need some tweaking, since they shouldn't have to go through sets 1-4 again. They'd just need to add a new commit to their existing branch and push it to their existing fork.

Reply:
Yes, that's what I mean. Steps 1-4 are creating a new commit and pushing. Should I just say add a new commit to their branch and push instead of referring them to the steps of how to do that?

Reply:
Maybe I should specify "Follow the steps 1-4 under Create a New Pull Request".

Reply:
I probably would.